integration_test/sql/sql.exs

```diff
@@ -152,3 +152,13 @@ defmodule Ecto.Integration.SQLTest do
     assert Ecto.Adapters.SQL.format_table(%{columns: ["test"], rows: nil}) == "+------+\n| test |\n+------+\n+------+"
   end
 end
+
+defmodule Ecto.Integration.SQLSyncTest do
+  use Ecto.Integration.Case, async: false
+
+  alias Ecto.Integration.PoolRepo
+
+  test "disconnect_all/2" do
+    assert :ok = PoolRepo.disconnect_all(0)
+  end
```

Comment:
Can probably merge it into the other test module? This file is meant to test eveerything exclusive to the SQL adapter anyway. `async: false`should not be necessary because disconnections should be "safe"?

Reply:
So... it shouldn't affect other tests because no other async test uses `PoolRepo`. But if they did, it could affect session level locks, for example. I wanted to go as safe as possible there, but I can move it up.

Reply:
Your call.